src/controllers/projects.js

```diff
@@ -0,0 +1,67 @@
+import Models from "../../models";
+
+const { Project } = models;
+
+const projectController = {
+  getSignedUrl: (req, res) => {
+    const s3 = new aws.S3();
+    const fileName = req.query["file-name"];
+    const fileType = req.query["file-type"];
+    const s3Params = {
+      Bucket: S3_BUCKET,
+      Key: fileName,
+      Expires: 60,
+      ContentType: fileType,
+      ACL: "public-read"
+    };
+
+    s3.getSignedUrl("putObject", s3Params, (err, data) => {
+      if (err) {
+        console.log(err);
+        return res.end();
+      }
+      const returnData = {
+        signedRequest: data,
+        url: `https://${S3_BUCKET}.s3.amazonaws.com/${fileName}`
+      };
+      res.write(JSON.stringify(returnData));
+      res.end();
+    });
+  },
+
+  postProject: async (req, res) => {
+    const { name, caption, photoUrl } = req.body;
+    const project = await Project.create({ name, caption, photoUrl });
+    res.status(201).send({ msg: "Project posted successfully", project });
+  },
```

Comment:
This keeps being undefined since the passed properties and the model are not the same.

> You are passing `name, caption , photoUrl` and your model has `title, description, thumbnail, projectUrl`.

Reply:
Nice catch